generators/client/templates/react/src/main/webapp/app/shared/reducers/application-profile.ts.ejs

```diff
@@ -49,5 +49,5 @@ export default (state: ApplicationProfileState = initialState, action): Applicat
 
 export const getProfile = () => ({
   type: ACTION_TYPES.GET_PROFILE,
-  payload: axios.get('/management/info')
```

Comment:
whats with the `register.ts.ejs` file change above?

Reply:
It was an unused and empty file so I removed it